ev3dev/core.py

```diff
@@ -61,15 +61,12 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
-# Keep the __version__ in sync with the package version in debian/changelog
```

Comment:
Because a4a296a creates ev3dev/version.py automatically. Less chances to forget to update this.